packages/menu-surface/index.tsx

```diff
@@ -128,13 +133,14 @@ class MenuSurface extends React.Component<MenuSurfaceProps, MenuSurfaceState> {
     if (quickOpen) {
       this.foundation.setQuickOpen(quickOpen);
     }
-    if (open) {
-      this.open_();
-    }
+    this.setState({mounted: true});
   }
 
-  componentDidUpdate(prevProps: MenuSurfaceProps) {
-    if (this.props.open !== prevProps.open) {
+  componentDidUpdate(prevProps: MenuSurfaceProps, prevState: MenuSurfaceState) {
+    // if this.props.open was true for the initial render
+    // then it will not be changed but the mounted state will be,
+    // so this.open_() should also be called in that case.
+    if (this.props.open !== prevProps.open || (this.props.open && this.state.mounted !== prevState.mounted)) {
```

Comment:
Please add a test for this new case that you just added

Reply:
Will do!